### PR TITLE
Remove GitHub colorscheme

### PR DESCRIPTION
**Pull request by jessieay**
* Every time I pull updates from this repo into my own dotfiles, I have to
  reconfigure my preferred theme (solarized)
* Seems like there is not a majority or plurality of thoughbotters using a
  single colorscheme
* Easiest to just not specify a colorscheme
* See https://forum.upcase.com/t/why-is-the-default-vim-theme-on-dotfiles-is-github/4232

### CODE REVIEW

File: vimrc [hunk 1]
```diff
@@ -75,7 +75,6 @@ if executable('ag')
 endif
 
 " Color scheme
-colorscheme github
 highlight NonText guibg=#060606
```
Comment (jferris) [inline vimrc]: Should we also take out these `highlight` preferences?

**Comment by jferris** [comment]
Good call! Should be easy to specify your own color scheme in `.local` files.

**Comment by croaky** [comment]
I use it, enjoy it, and am just as happy to have it in my personal dotfiles rather than thoughtbot dotfiles. Tested with `.local` and works as expected. Looks good to merge.

File: vimrc (hunk 1 above)
Comment (jessieay) [inline vimrc]: I think this is for highlighting extra whitespace, which I think is good for any colorscheme. Reference https://wincent.com/blog/making-vim-highlight-suspicious-characters

Thoughts?

Comment (derekprior) [inline vimrc]: I think it's a good idea, but we don't know what background people are using and can't be sure this will be an affective highlight.

Comment (gylaz) [inline vimrc]: I didn't realize trailing whitespace was highlighted, likely due to my dark color-scheme.

File: vimrc [hunk 2]
```diff
@@ -75,7 +75,6 @@ if executable('ag')
...
 highlight Folded  guibg=#0A0A0A guifg=#9090D0
```
Comment (gylaz) [inline vimrc]: Should we remove this too?

File: vimrc (hunk 1 above)
Comment (jessieay) [inline vimrc]: yeah, same actually. Will remove.

**Comment by jessieay** [comment]
@croaky added the local color setup to README - re-review?

**Comment by croaky** [comment]
Looks good!

**Pull request merged**